CIP-0030/README.md

```diff
@@ -81,35 +81,74 @@ transaction_unspent_output = [
 
 then we define
 
-```
+```cddl
 type TransactionUnspentOutput = cbor<transaction_unspent_output>
 ```
 
 This allows us to use the output for constructing new transactions using it as an output as the `transaction_output` in the Shelley Multi-asset CDDL does not contain enough information on its own to spend it.
 
 #### Paginate
 
-```
+```ts
 type Paginate = {|
   page: number,
   limit: number,
 |};
 ```
+
 Used to specify optional pagination for some API calls. Limits results to {limit} each page, and uses a 0-indexing {page} to refer to which of those pages of {limit} items each. dApps should be aware that if a wallet is modified between paginated calls that this will change the pagination, e.g. some results skipped or showing up multiple times but otherwise the wallet must respect the pagination order.
 
 #### Extension
```

Comment:
IMO it would make extension discovery easier if we have a list of accepted extensions here in the CIP or an attached file to make it machine readable?